mmf/datasets/databases/readers/feature_readers.py

```diff
@@ -57,7 +57,7 @@ def _init_reader(self):
         elif self.ndim == 3 and not self.depth_first:
             self.feat_reader = Dim3FeatureReader()
         elif self.ndim == 4 and self.depth_first:
-            self.feat_reader = CHWFeatureReader()
+            self.feat_reader = CHWFeatureReader(self.max_features)
```

Comment:
Would it be not possible to use PaddedFasterRCNN or refactor this logic of padding?

Reply:
The features from `grid-feat-vqa` are of CHW format and doesn't have any `_info` files that are used in PaddedFasterRCNN. So I would keep those two separate.